bitnami/wordpress-nginx/README.md

```diff
@@ -240,6 +240,7 @@ Available environment variables:
 - `WORDPRESS_SKIP_BOOTSTRAP`: Skip the WordPress installation wizard. This is necessary when providing a database with existing WordPress data. Default: **no**
 - `WORDPRESS_AUTO_UPDATE_LEVEL`: Level of auto-updates to allow for the WordPress core installation. Valid values: `major`, `minor`, `none`. Default: **none**
 - `WORDPRESS_ENABLE_REVERSE_PROXY`: Enable WordPress support for reverse proxy headers. Default: **no**
+- `WORDPRESS_ENABLE_XML_RPC`: Enable the WordPress XML-RPC endpoint. Default: **yes**
```

Comment:
I do believe that this point should be disabled by default. I would change the default value to `no` and add a note in the [Notable Changes](https://github.com/bitnami/containers/blob/main/bitnami/wordpress/README.md#notable-changes) section regarding this change.

If someone is using is using it, they can simply turn the feature back by tuning the value. WDYT?

Reply:
Sure, this is fine for me. 👍